linera-storage/src/lib.rs

```diff
@@ -321,20 +316,56 @@ where
     }
 
     async fn write_certificate(&self, certificate: &Certificate) -> Result<(), ViewError> {
+        let mut batch = Batch::new();
+        self.write_certificate_batch(certificate, &mut batch)?;
+        self.write_batch(batch).await
+    }
+
+    async fn write_certificates(&self, certificates: &[Certificate]) -> Result<(), ViewError> {
+        let mut batch = Batch::new();
+        for certificate in certificates {
+            self.write_certificate_batch(certificate, &mut batch)?;
+        }
+        self.write_batch(batch).await
+    }
+
+    fn wasm_runtime(&self) -> Option<WasmRuntime> {
+        self.client.wasm_runtime
+    }
+}
+
+impl<CL> DbStoreClient<CL>
+where
+    CL: KeyValueStoreClient + Clone + Send + Sync + 'static,
+    ViewError: From<<CL as KeyValueStoreClient>::Error>,
+    <CL as KeyValueStoreClient>::Error: From<bcs::Error> + Send + Sync + serde::ser::StdError,
+{
+    fn write_value_batch(&self, value: &HashedValue, batch: &mut Batch) -> Result<(), ViewError> {
```

Comment:
These new `write_xxx_batch` look like variants of `write_batch` which is a bit confusing.
Maybe `add_xxx_to_batch` ?